content/blog/2020-12-tokio-1-0.md

```diff
@@ -0,0 +1,150 @@
+---
+date: "2020-12-23"
+title: "Announcing Tokio 1.0"
+description: "December 23, 2020"
+---
+
+We are pleased to announce the 1.0 release of Tokio, an asynchronous runtime for
+the Rust programming language. Tokio provides the needed building blocks for
+writing reliable networking applications without compromising speed. It comes
+with asynchronous APIs for TCP, UDP, timers, a [multi-threaded, work-stealing
+scheduler][sched], and more.
+
+[sched]: https://tokio.rs/blog/2019-10-scheduler
+
+Over the years, we have been delighted watching our users build amazing things.
+For example, [Discord] used Tokio to reduce tail latencies 5x. [Fly.io] found
+that, with Tokio, they can effortlessly meet their performance requirements and
+focus on delivering features to their customers. For [ZCash], building on Tokio
+enables them to design misuse-resistant APIs. For AWS, their [Lambda][lambda]
+team used Tokio to implement a more reliable and flexible version of
+their stateful router.
+
+[Discord]: https://blog.discord.com/why-discord-is-switching-from-go-to-rust-a190bbca2b1f
+[Fly.io]: https://fly.io
+[ZCash]: https://www.zfnd.org/blog/futures-batch-verification/
+[lambda]: https://aws.amazon.com/lambda/
+
+The [tutorial] is the best place to learn how to get started with Tokio.
+
+[tutorial]: https://tokio.rs/tokio/tutorial
+
+# A stability guarantee
+
+We first [announced] Tokio over four years ago. Since then, it has evolved
+significantly. The most notable change happened a year ago with the addition of
+[async and await][async_await] to Rust. Today, Tokio is easier to use and more
+powerful. This evolution has also caused some friction. It has required
+libraries to track these changes. It could also result in confusing error
+messages when accidentally depending on multiple versions of Tokio.
+
+[announced]: https://medium.com/@carllerche/announcing-tokio-df6bb4ddb34
+[async_await]: https://tokio.rs/blog/2019-11-tokio-0-2
+
+The Tokio 1.0 release ends this churn. As part of the release, we are committing
+to providing a stable foundation for the ecosystem. We have no current plans for
+Tokio 2.0 and we commit to holding back on a Tokio 2.0 release for at least 3
+years. We also commit to maintain the Tokio 1.0 branch for a minimum of 5 years.
+Tokio will keep a rolling MSRV (minimum supported rust version) policy of 6
+months. When increasing the MSRV, the new Rust version must have been released
+at least six months ago.
+
+This stability does not mean that Tokio will stagnate. We still have a lot of
```

Comment:
```suggestion
This stability does not mean Tokio will stagnate. We still have a lot of
```

Reply:
I think it's more natural to keep "that" here.

Reply:
My criteria is if removing it doesn't hurt legibility. But I'm fine either way!